### ISSUE TITLE
In double-qouted string variable: Parse Error

### ISSUE THREAD

**Issue by jens1o** [Microsoft/tolerant-php-parser#219]
See following file:
```php
<?php

namespace Illuminate\Database\Connectors;

use InvalidArgumentException;

class SQLiteConnector extends Connector implements ConnectorInterface
{
    /**
     * Establish a database connection.
     *
     * @param  array  $config
     * @return \PDO
     *
     * @throws \InvalidArgumentException
     */
    public function connect(array $config)
    {
        $options = $this->getOptions($config);

        // SQLite supports "in-memory" databases that only last as long as the owning
        // connection does. These are useful for tests or for short lifetime store
        // querying. In-memory databases may only have a single open connection.
        if ($config['database'] == ':memory:') {
            return $this->createConnection('sqlite::memory:', $config, $options);
        }

        $path = realpath($config['database']);

        // Here we'll verify that the SQLite database exists before going any further
        // as the developer probably wants to know if the database exists and this
        // SQLite driver will not throw any exception if it does not by default.
        if ($path === false) {
            throw new InvalidArgumentException("Database (${config['database']}) does not exist.");
        }

        return $this->createConnection("sqlite:{$path}", $config, $options);
    }
}
```

I get this parse error:

![image](https://user-images.githubusercontent.com/11234139/35183376-4718c388-fde5-11e7-9363-1e24e7c79473.png)

(Yes the issue title is inappropriate)

**Comment by mattacosta** [comment]
Appears to be caused by the same thing as #214.

**Comment by jens1o** [comment]
Could be, but I'm not sure. Perhaps somebody knows more?

**Comment by TysonAndre** [comment]
This should be fixed in tolerant-php-parser 0.0.9 by https://github.com/Microsoft/tolerant-php-parser/pull/226

**Comment by jens1o** [comment]
Yeah, this is fixed. Thank you @TysonAndre 
![image](https://user-images.githubusercontent.com/11234139/36296957-3fcd8f76-12f0-11e8-8d99-f2044511ebe7.png)
